docs/cmd/kn_service_migrate.md

```diff
@@ -0,0 +1,52 @@
+## kn service migrate
+
+Migrate Knative services from source cluster to destination cluster
+
+### Synopsis
+
+Migrate Knative services from source cluster to destination cluster
+
+```
+kn service migrate [flags]
+```
+
+### Examples
+
+```
+
+  # Migrate Knative services from source cluster to destination cluster by export KUBECONFIG and KUBECONFIG_DESTINATION as environment variables
```

Comment:
Let's remove the KUBECONFIG_DESTINATION env var, as it's not really something you want to set-and-forget